src/main/java/generator/BaseScenarioGenerator.java

```diff
@@ -115,6 +117,8 @@ private void addPedestrian(Model model) {
         person.addDistance(3000F);
 //        person.addValueInDollars(10000F);
         Lane lane = model.getLanes().get(Model.Side.CENTER).get(0);
+        person.addIs_on_lane(lane);
+        lane.addLane_has_pedestrian(person);
```

Comment:
Tak samo jak w przypadku vehicle, ustawianie propertisów

Reply:
od naszej strony to jest ok i wydaje się być sensowne, więc ja nie mam uwag